client/reader/following-edit/index.jsx

```diff
@@ -544,7 +545,12 @@ const FollowingEdit = React.createClass( {
 							? <FollowingEditSortControls onSelectChange={ this.handleSortOrderChange } sortOrder={ this.state.sortOrder } />
 							: null }
 					{ ! hasNoSubscriptions
-							? <Gridicon icon="search" className="following-edit__search" onClick={ this.toggleSearching } />
+							? <Button 
```

Comment:
Shall we add an `aria-label` to the button too? I see the `Search` component has:

`aria-label={ i18n.translate( 'Open Search', { context: 'button label' } ) }`